Makefile

```diff
@@ -229,14 +229,23 @@ install-resources:
 e2e-dependencies:
 
 K8SCLIENT ?= oc
+GINKGO = $(LOCAL_BIN)/ginkgo
+IS_HOSTED ?= false
+CLUSTER_NAMESPACE_ON_HUB =
 .PHONY: e2e-test
 e2e-test:
 	@if [ -z "$(TEST_FILE)" ]; then\
-		$(GINKGO) -v --no-color $(TEST_ARGS) --fail-fast test/e2e -- -cluster_namespace=$(MANAGED_CLUSTER_NAME) -k8s_client=$(K8SCLIENT) ;\
+		$(GINKGO) -v --no-color $(TEST_ARGS) --fail-fast test/e2e -- -cluster_namespace=$(MANAGED_CLUSTER_NAME) -k8s_client=$(K8SCLIENT) -is_hosted=$(IS_HOSTED) -cluster_namespace_on_hub=$(CLUSTER_NAMESPACE_ON_HUB);\
```

Comment:
Could you use a new variable called `MANAGED_CLUSTER_NAMESPACE` and have it default to `MANAGED_CLUSTER_NAME`? I find it confusing that `MANAGED_CLUSTER_NAME` is set to `cluster2-hosted` because its actual name is still `cluster2`, it's just its installed namespace on the hosting cluster is `cluster2-hosted`.